code/src/App.js

```diff
@@ -1,9 +1,53 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, createStore } from '@reduxjs/toolkit'
+
+import Header from 'components/Header'
+import AddQuest from 'components/AddQuest'
+import QuestList from 'components/QuestList'
+import Progress from 'components/Progress'
+import Footer from 'components/Footer'
+import quests from './reducers/quests'
+
+const reducer = combineReducers({
+  quests: quests.reducer,
+})
+
+// get the localStorage in JSON string format
+const persistedStateJSON = localStorage.getItem('questsReduxState')
+let persistedState = {}
+
+// if we have data in persistedStateJSON -> convert to an object
+if (persistedStateJSON) {
+  persistedState = JSON.parse(persistedStateJSON)
+}
+
+// create a store with initial state
+const store = createStore(
+  reducer,
+  persistedState,
+  /* enable the redux devtools */
+  /* eslint-disable-next-line no-underscore-dangle */
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+)
+
+// update the title with the amount of quests
+document.title = `(${store.getState().quests.items.length}) My Quests`
```

Comment:
Is this the syntax that update the browser tab with the todos? That was a nice touch!